build.sbt

```diff
@@ -42,7 +42,7 @@ lazy val geohex = crossProject.in(file(".")).
   settings(
     organization := "net.teralytics",
     name := "geohex",
-    version := "0.1." + sys.env.getOrElse("TRAVIS_BUILD_NUMBER", "0-SNAPSHOT"),
+    version := "0.2." + sys.env.getOrElse("TRAVIS_BUILD_NUMBER", "0-SNAPSHOT"),
```

Comment:
At this point, API of the library is in complete flux and every change is potentially breaking. Especially around JS/JVM interop. I'd refrain from giving a false sense of compatibility and stick to `0.1.x` for now.